.github/workflows/ci.yml

```diff
@@ -16,5 +16,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@8c0edbc76e98fa90f69d9a2c020dcb50019dc325
+        with:
+          install: true
```

Comment:
This ensures that when calling `docker build` in our `bin/run-tests-in-docker.sh` script, it will actually use `docker buildx` instead.
This allows us to transparently use caching without referencing buildx in our scripts.